### PR TITLE
AAP-30371: user reports: add extra columns

### PR DESCRIPTION
**Pull request by goneri**
Add:
- Org id
- Org has_api_key
- User UUID
- Trial end (date)

### CODE REVIEW

File: ansible_ai_connect/users/reports/tests/test_generators.py [hunk 1]
```diff
@@ -143,11 +154,19 @@ def tearDown(self):
...
-        return "First name,Last name,Organization name,Plan name,Trial started"
+        return (
+            "OrgId,Org has_api_key,UUID,First name,Last name,Organization name,"
+            "Plan name,Trial started,Trial expired_at"
+        )
 
     def get_report_generator(self) -> BaseGenerator:
         return UserTrialsReportGenerator()
 
+    def test_get_report_in_the_right_order(self):
```
Comment (manstis) [inline ansible_ai_connect/users/reports/tests/test_generators.py:165]: Arguably this doesn't test ordering.

**Review by manstis — DISMISSED** [review]
LGTM 👍 

One of the new tests is questionable.

File: ansible_ai_connect/users/reports/tests/test_generators.py (hunk 1 above)
Comment (goneri) [reply ansible_ai_connect/users/reports/tests/test_generators.py:165]: Agreed, I will improve that.

**Review by manstis — APPROVED** [review]
Thanks. Nice test. LGTM 👍

**Pull request merged**